crates/core/tedge_agent/src/software_manager/actor.rs

```diff
@@ -33,12 +32,6 @@ use tedge_config::TEdgeConfigError;
 use tracing::error;
 use tracing::info;
 use tracing::warn;
-use which::which;
-
-#[cfg(not(test))]
-const SUDO: &str = "sudo";
-#[cfg(test)]
-const SUDO: &str = "echo";
 
```

Comment:
This was actually used by some tests. Notably `tedge-agent` `test_new_software_update_operation`.